ts/kpt-functions/src/types.ts

```diff
@@ -386,6 +386,14 @@ export interface Result {
  */
 export type Severity = 'error' | 'warn' | 'info';
 
+interface JsonArray extends Array<Json> {}
+
+interface JsonObject {
```

Comment:
Can we rename this to JsonMap?

Reply:
Sure, just noting that this is actually called `object` in the JSON spec.

Reply:
I still think JsonMap is clearer but don't have a strong opinion either way.